### PR TITLE
Ashley/onboarding

### PR DESCRIPTION
**Pull request by ashleynguyen27**
## OnBoarding

Backbone of onbaording

### Related PRs

Optional - any related PRs you're waiting on, or PRs that will conflict, etc

### Migrations

Optional - if you added anything to the database through migration(s)

### Tests Performed, Edge Cases

Lol we'll see if we test

### Screenshots

Add screenshots!!! If you'd like

CC: @aivantg

### CODE REVIEW

File: src/onboarding.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                        errorMessage = value.matches(/(?=.*[0-9])/) ? '' : 'Password must include a number';
+                    }
+            }
+        }
+        this.setState({
+            errors : {...this.state.errors, [event.target.name]: errorMessage}
+        })
+    }
+
+    handleClick(){
```
Comment (aivantg) [inline src/onboarding.js]: can't this be combined with `handleChange`?

File: src/onboarding.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                beneficiaries: [], //because there can be multiple
+                payment_info: "", //need to revisit**
+                b_street: "",
+                b_apt: "",
+                b_state: "",
+                b_zipcode: ""
+            },
+            step: 1
+        };
+        this.handleChange = this.handleChange.bind(this);
```
[the remark this answers is not in the document]
Comment (ashleynguyen27) [reply src/onboarding.js]: i actually do not know the difference between binding and the arrow lol. pls

File: src/onboarding.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+        this.setState({step: step + 1});
+    }
+
+    //prev function decrements page down one and switches to that numbered page
+    prevStep = () => {
+        const { step } = this.state
+        this.setState({step: step - 1});
+    }
+
+    callBackBylawValidation() {
```
[the remark this answers is not in the document]
Comment (ashleynguyen27) [reply src/onboarding.js]: for some reason it didnt work in the handleFormValidation function but it did on its own. I have no idea

File: src/onboarding.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+                })
+            default:
+                this.setState({
+                    [event.target.name] : event.target.value
+                })
+        }
+    }
+
+    //validates the input divs
+    handleFormValidation = event => {
```
[the remark this answers is not in the document]
Comment (ashleynguyen27) [reply src/onboarding.js]: decided to use this instead of the formValidation

File: src/onboarding.js (hunk 1 above)
Comment (ashleynguyen27) [reply src/onboarding.js]: for some reason this worked on its own too but not in handlechange. i have no idea.  wanted to revisit this with you when u were free
